website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -13,6 +13,7 @@ You can create conditions that produce custom error messages for several types o
 This page explains the following:
   - Creating [validation conditions](#input-variable-validation) for input variables (Terraform v0.13.0 and later)
   - Creating [preconditions and postconditions](#preconditions-and-postconditions) for resources, data sources, and outputs (Terraform v1.2.0 and later)
+  - Creating [assertions](#check-assertions) within check blocks (Terraform v1.5.0 and later)
   - Writing effective [condition expressions](#condition-expressions) and [error messages](#error-messages)
   - When Terraform [evaluates custom conditions](#conditions-checked-only-during-apply) during the plan and apply cycle
 
```

Comment:
Below here, I think it'd be nice to have a paragraph clarifying when users should use which kind of custom condition. Especially since folks can do things in difference ways, we want **strong** guidance at the beginning so folks don't have to read this whole page to figure out which kind they should use (checks vs. pre/post conditions .v.s. validation blocks). 

This paragraph doesn't need to describe how the feature works, we can link to sections below.